systemtests/scripts/functions

```diff
@@ -1121,6 +1131,7 @@ touch "${tmp}/dir.out" "${tmp}/fd.out" "${tmp}/sd.out"
 
 HOST=${HOST:-bareos}
 CLIENT="${HOST}-fd"
+USER=${USER:-$(whoami)}
```

Comment:
this is a leftover from your first commit, right?

Reply:
I forget to remove it, well catch, now fixed